libexec/rtld-elf/rtld_c18n.c

```diff
@@ -142,7 +142,7 @@ static struct compart rtld_compart = {
 };
 
 static struct compart libthr_compart = {
-	.name = "libthr.so.3",
+	.name = "[TCB]",
```

Comment:
I mean it's still called libthr_compart... and is this really needed as part of the bug fix?

Reply:
Good catch. This actually fixes another tracing bug where transitions to libc are being printed as transitions to libthr. I'll mention it in the commit message.